### ISSUE TITLE
force_logger defaults to log_level :auto instead of inheriting from log_level in client.rb

### ISSUE THREAD

**Issue by vinyar** [chef/chef#3770]
Bug: setting force_logger to true produces STDOUT with almost no output (log level warn/auto). Output level can not be modified.

Expected behavior: force_logger true STDOUT output is at the same level as output piped to the log file.

source code:
https://github.com/chef/chef/blob/master/chef-config/lib/chef-config/config.rb#L271-L272
https://github.com/chef/chef/blob/master/lib/chef/application.rb#L148

client.rb

``` ruby
log_level                :info
log_location             '/etc/chef/trash.txt'
force_logger        true
deamonize       false
```

chef-shell:

``` ruby
chef > Chef::Log.init(MonoLogger.new(Chef::Config[:log_location]))
 => #<MonoLogger:0x007f9473b4c968 @progname=nil, @level=2, @default_formatter=#<Logger::Formatter:0x007f9473b4c918 @datetime_format=nil>, @formatter=#<Mixlib::Log::Formatter:0x007f9473b4c738 @datetime_format=nil>, @logdev=#<MonoLogger::LocklessLogDevice:0x007f9473b4c8c8 @shift_size=nil, @shift_age=nil, @filename=nil, @dev=#<IO:<STDOUT>>>>
chef > Chef::Log.level
 => :warn

stdout_logger = MonoLogger.new(STDOUT)
 => #<MonoLogger:0x007f9473a5e650 @progname=nil, @level=0, @default_formatter=#<Logger::Formatter:0x007f9473a5e628 @datetime_format=nil>, @formatter=nil, @logdev=#<MonoLogger::LocklessLogDevice:0x007f9473a5e600 @shift_size=nil, @shift_age=nil, @filename=nil, @dev=#<IO:<STDOUT>>>>
chef > stdout_logger.level
 => 0
```

linked doc bug: https://github.com/chef/chef-docs/issues/612

**Comment by vinyar** [comment]
possible workaround (untested) - `chef-client -z -o etc. etc. | tee -a /my/log/file.log

**Comment by lamont-granquist** [comment]
Probably related to #2514 and should be fixed when that gets fixed at the very least.

**Comment by lamont-granquist** [comment]
not sure if this is still relevant but it needs to be addressed as part of #9126
